torchvision/prototype/transforms/functional/_color.py

```diff
@@ -289,7 +295,21 @@ def adjust_gamma(inpt: features.InputTypeJIT, gamma: float, gain: float = 1) ->
         return adjust_gamma_image_pil(inpt, gamma=gamma, gain=gain)
 
 
-posterize_image_tensor = _FT.posterize
+def posterize_image_tensor(image: torch.Tensor, bits: int) -> torch.Tensor:
+    if image.is_floating_point():
+        levels = 1 << bits
+        return image.mul(levels).floor_().clamp_(0, levels - 1).div_(levels)
+    else:
+        num_value_bits = _num_value_bits(image.dtype)
+        if bits < num_value_bits:
+            mask = ((1 << bits) - 1) << (num_value_bits - bits)
+            return image & mask
+        elif bits == num_value_bits:
+            return image
+        else:
+            raise ValueError
```

Comment:
Instead of raising here, we could simply return. Meaning trying to posterize an uint8 image with 8 or more bits does nothing as is expected. Basically we need to decide whether we consider this a user error or not.

Reply:
I would be returning the image but what you have here also works. Up to you.

Reply:
I'll remove the raise.